sinks_test.go

```diff
@@ -166,6 +172,61 @@ func TestSHA256Sum(t *testing.T) {
 	}
 }
 
+func TestSliceSink(t *testing.T) {
+	tests := []struct {
+		name    string
+		fields  *Pipe
+		want    []string
+		wantErr bool
+	}{
+		{
+			name: "Multiple lines pipe",
+			fields: Echo("testdata/multiple_files/1.txt\ntestdata/multiple_files/2.txt\ntestdata/multiple_files/3.tar.zip\n"),
+			want: []string{
+				"testdata/multiple_files/1.txt",
+				"testdata/multiple_files/2.txt",
+				"testdata/multiple_files/3.tar.zip",
+			},
+			wantErr: false,
+		},
+		{
+			name: "Empty pipe",
+			fields: Echo(""),
+			want: []string{},
+			wantErr: false,
+		},
+		{
+			name: "Single newline",
+			fields: Echo("\n"),
+			want: []string{""},
+			wantErr: false,
+		},
+		{
+			name: "Empty line between two existing lines",
+			fields: Echo("testdata/multiple_files/1.txt\n\ntestdata/multiple_files/3.tar.zip"),
+			want: []string{
+				"testdata/multiple_files/1.txt",
+				"",
+				"testdata/multiple_files/3.tar.zip",
+			},
+			wantErr: false,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			p := tt.fields
+			got, err := p.Slice()
+			if (err != nil) != tt.wantErr {
+				t.Errorf("Slice() error = %v, wantErr %v", err, tt.wantErr)
+				return
+			}
+			if !cmp.Equal(got, tt.want) {
```

Comment:
I like to always compare in the order `want, got`, because when I print the diff, that's the order that makes sense. And though it makes no difference to Go, it's pleasing to me to have them in the same order everywhere!